### PR TITLE
Done

### PR DESCRIPTION
**Pull request by AlexandreMarquis31**

### CODE REVIEW

File: pandas_questions.py [hunk 1]
```diff
@@ -28,28 +27,36 @@ def merge_regions_and_departments(regions, departments):
...
-def merge_referendum_and_areas(referendum, regions_and_departments):
+    d = pd.DataFrame({
+        'code_dep': departments["code"],
+        'name_dep': departments["name"],
+        'code_reg': departments["region_code"]
+    })
+    d2 = pd.DataFrame({
+        'name_reg': regions["name"],
+        'code_reg': regions["code"]
+    })
```
Comment (tomMoral) [inline pandas_questions.py:38]: Rmk: Note that you can use `rename` to simply rename some columns.

Also, the renaming can be done directly with `suffixes` in merge.

**Review by tomMoral — APPROVED** [review]
LGTM!

**Comment by tomMoral** [comment]
If you are finished, you can sign up for this RAMP challenge: https://ramp.studio/events/brevet_datacamp2020
This is what we are going to do this afternoon. The goal is to try and solve the challenge.
You need to click on the blue button: `Join the event` and I need to approve your participation so send me a message once you have done this. If you don't see the join button, please click here: https://ramp.studio/events/brevet_datacamp2020/sign_up
You can then read the starting kit and go to the left menu to submit your solutions in the sandbox.

File: pandas_questions.py (hunk 1 above)
Comment (AlexandreMarquis31) [reply pandas_questions.py:38]: Ah yes, forgot about this, thank you !

**Pull request closed without merging**